src/test/fuzz/fuzz.cpp

```diff
@@ -27,8 +32,16 @@ void FuzzFrameworkRegisterTarget(std::string_view name, TypeTestOneInput target,
 
 static TypeTestOneInput* g_test_one_input{nullptr};
 
+static std::unique_ptr<Sock> TerminateOnTCPSocketCreation(const CService&)
+{
+    std::terminate();
+}
+
```

Comment:
Can be removed if done with a simple one-line lambda:

```diff
diff --git a/src/test/fuzz/fuzz.cpp b/src/test/fuzz/fuzz.cpp
index b6ba44fe21..631c861bb6 100644
--- a/src/test/fuzz/fuzz.cpp
+++ b/src/test/fuzz/fuzz.cpp
@@ -32,15 +32,10 @@ void FuzzFrameworkRegisterTarget(std::string_view name, TypeTestOneInput target,
 
 static TypeTestOneInput* g_test_one_input{nullptr};
 
-static std::unique_ptr<Sock> TerminateOnTCPSocketCreation(const CService&)
-{
-    std::terminate();
-}
-
 void initialize()
 {
     // Terminate immediately if a fuzzing harness ever tries to create a TCP socket.
-    CreateSock = TerminateOnTCPSocketCreation;
+    CreateSock = [](const CService&) -> std::unique_ptr<Sock> { std::terminate(); };
 
     bool should_abort{false};
     if (std::getenv("PRINT_ALL_FUZZ_TARGETS_AND_ABORT")) {

Reply:
Sure, why not! Now addressed :)